test/common_plat/common/mask_common.c

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+#include "config.h"
```

Comment:
I suppose this config.h will also be included in the distribution package, right?
is that possible to include "config.h" in global odp_api.h?
So applications only need to include the "odp_api.h" and implicitly include the "config.h"

I check the approach of DPDK, which has a similar rte_config.h, included by rte_eal.h, so applications would need to include "rte_config.h" explicitly.